apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.service.js

```diff
@@ -110,3 +115,125 @@ export async function unPublish(id) {
 		published: false
 	});
 }
+
+/**
+ *
+ * @param {import('@pins/applications.api').Schema.ExaminationTimetableItem} examinationTimetableItem
+ * @param {Number} parentFolderId
+ * @param {Number} caseId
+ * @returns
+ */
+export const createDeadlineSubFolders = async (
+	examinationTimetableItem,
+	parentFolderId,
+	caseId
+) => {
+	if (!examinationTimetableItem?.description) {
+		return;
+	}
+
+	const description = JSON.parse(examinationTimetableItem?.description);
```

Comment:
I thought we were creating individual entries for each line item on creation (rather than just creating the folders), this is what the PINS Data Model expects anyway (it expects a eventLineItems array with unique IDs, and those IDs would be how the front office communicates submissions to us). I guess with this there'd be no concept of line items outside of strings in the description.

Does this mean that for submissions we're looking up the folder by name?

Reply:
I'm not sure how it works currently, I'd have to dig into the code a bit more. This isn't actually new code, I've just moved it into service from controller. If you look in `examination-timetable-items.controller.js` on `main` the same code is there.

Reply:
Ah apologies I didn't see the delete - I'll move this discussion to the teams chat

Reply:
The first commit is all just restructuring. Everything feature-relevant is in the following commits :)